### PR TITLE
Add `style` package export for dist/style.css

### PR DESCRIPTION
**Pull request by kerryarchibald**
The following syntax will set the style condition:

- CSS @import
- HTML <link rel="stylesheet">

### CODE REVIEW

**Review by dbkr — APPROVED** [review]
In wasn't aware of this but looks like it's a thing that webpack reads (https://webpack.js.org/guides/package-exports/#conditions)? Looks fine. What happened if the style export wasn't set? It failed rather than falling back to one of the others?

**Comment by kerryarchibald** [comment]
> In wasn't aware of this but looks like it's a thing that webpack reads (https://webpack.js.org/guides/package-exports/#conditions)? Looks fine. What happened if the style export wasn't set? It failed rather than falling back to one of the others?

Yep, when the specific type is not specified it will result in module not found:

> When the exports field is specified, only these module requests are available. Any other requests will lead to a ModuleNotFound Error.

File: package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
     },
     "./dist/style.css": {
       "require": "./dist/style.css",
-      "import": "./dist/style.css"
+      "import": "./dist/style.css",
+      "style": "./dist/style.css"
```
Comment (sandhose) [inline package.json:17]: Wouldn't it make sense then to set it in the `.` import? I think that would make `@import "@vector-im/compound-web"` work?

Comment (kerryarchibald) [reply package.json]: Oh good idea!

**Pull request merged**